homeassistant/components/daikin/config_flow.py

```diff
@@ -75,7 +75,8 @@ async def _create_device(self, host, key=None, password=None):
                     uuid=uuid,
                     password=password,
                 )
-        except asyncio.TimeoutError:
+        except (asyncio.TimeoutError, ClientConnectorError):
```

Comment:
`ClientConnectorError` is actually derived eventually from ClientError, which is already catched below.

Isn't it more logical to change it to:

```py
except (asyncio.TimeoutError, ClientError):
    self.host = None
    return self.async_show_form(
        step_id="user",
        data_schema=self.schema,
        errors={"base": "cannot_connect"},
    )
```

And remove the `ClientError` catch below (that now throws a `unknown` result error)

Reply:
Good points, `ClientError` could be so much more than just a connection error that is why I went this way. It might be easier to trouble shoot unforeseen issues if we go my way, but your way is of course less code. 

- [ ] keep @fredrike's code
- [ ] go with @frenck's suggestion

Reply:
From a user perspective, if a client error is thrown; you can safely report it could not connect. Sure, there can be other things going on, but the result towards the end-user would be the same: cannot connect.